### ISSUE TITLE
Error : failed copying 'chain.c32'

### ISSUE THREAD

**Issue by mohit000001** [emmett1/lfs-scripts#36]
I have created and complied all the lfs things using their book (lfs 11.01). Now i was using 03-mkiso of yours script to create an iso image for my lfs system but getting following error.

:: Preparing isolinux files...
cp: cannot stat '/mnt/lfs/usr/share/syslinux/chain.c32': No such file or directory
ERROR: failed copying 'chain.c32'

Please provide solution for this.

**Comment by emmett1** [comment]
Do you install grub into your lfs?

**Comment by mohit000001** [comment]
@emmett1 yes , used  following as per the lfs book.
 "grub-install /dev/sda "

cat > /boot/grub/grub.cfg << "EOF"
set default=0
set timeout=5
insmod ext2
set root=(hd0,msdos3)
menuentry "GNU/Linux, Linux 5.16.9-lfs-11.1" {
 linux /boot/vmlinuz-5.16.9-lfs-11.1 root=/dev/sda3 ro
}
EOF

**Comment by emmett1** [comment]
No, i mean do you install grub into your lfs system? Not generate grub config.

**Comment by mohit000001** [comment]
Yes I do install grub package. It is in list of lfs packages of chapter 8.  chapter 8.59  - GRUB-2.06. I have installed that in lfs.
